proposals/0293-vehicle-type-filter.md

```diff
@@ -266,22 +143,215 @@ The Android proxy will need to implement the above protocol changes. In addition
 #### Metadata for Android Router Service
 
 1. This resource file needs to be referenced in the manifest file of the project as `metadata` for Android Router Service.
-2. When the proxy receives `GetVehicleTypeACK`, it will check if the vehicle type information is defined in the provided supported vehicle type list.
+2. When the proxy receives `StartServiceACK` with vehicle type details, it will check if the vehicle type information is defined in the provided supported vehicle type list.
```

Comment:
```suggestion
2. When the app library receives `StartServiceACK` with vehicle type details, it will check if the vehicle type information is defined in the provided supported vehicle type list.
```